contracts/collections/sg721-base/src/contract.rs

```diff
@@ -240,7 +269,7 @@ where
         info: MessageInfo,
         start_time: Option<Timestamp>,
     ) -> Result<Response, ContractError> {
-        let minter = self.parent.minter.load(deps.storage)?;
+        let minter = self.minter.load(deps.storage)?;
         if minter != info.sender {
             return Err(ContractError::Unauthorized {});
         }
```

Comment:
this needs to check cw_owner instead of self.minter

Reply:
This has been addressed by using    ```let owner = cw_ownable::get_ownership(deps.storage);```